src/pages/Components/FileUploader/code.md

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Code'
+order: 2
+---
+## Import
+
+Import the settings and the checkbox `scss` files.
+
+```css
+@import 'settings-tools/_all-settings';
+@import 'components/_c.file-uploader';
+```
+
+## Basic usage
+> Natively, to upload files we use an `<input type="file"/>`, but in Mozaic, we will customize it with a `<label>` that will act as a button.
+
+Wrap a input[type="file"] and a label inside a div and apply the following classes :
+
+```html
+<div class="mc-fileuploader">
+    <input type="file" aria-label="Click to choose a file on your device" class="mc-fileuploader__input" id="import-file">
+    <label for="import-file" class="mc-fileuploader__label"><span>file uploader</span></label>  
+</div>
+```
+<preview path="src/pages/Components/FileUploader/previews/file-uploader"></preview>
```

Comment:
Can we change the label of the button like "Select the file to upload" ?
The label is an important part of this type of components, I think it's better to put a right label for the comprehension and the use of the pattern